R/imputation.R

```diff
@@ -272,53 +305,95 @@ imputeMethods <- function()
 ##' @param k `numeric(1)` providing the imputation value used for the
 ##'     first and last samples if they contain an `NA`. The default is
 ##'     to use the smallest value in the data.
-impute_neighbour_average <- function(x, k = min(x, na.rm = TRUE)) {
+impute_neighbour_average <- function(x, k = min(x, na.rm = TRUE), margin = 1L) {
```

Comment:
Just a question - would it maybe be better to call that parameter `MARGIN` instead? Such as in `apply`?

Reply:
OK, will change it and then push to Bioc.